packages/config/package.json

```diff
@@ -43,7 +43,7 @@
         "util": "^0.12.4",
         "webpack": "^5.38.1",
         "webpack-cli": "^4.7.2",
-        "webpack-dev-server": "^3.11.2",
+        "webpack-dev-server": "^4.2.1",
```

Comment:
There is lot of changes, so it's not that simple: https://github.com/webpack/webpack-dev-server/blob/master/migration-v4.md 😭 

cc @redallen